includes/frontend/class-wcmp-frontend.php

```diff
@@ -223,6 +229,16 @@ public static function getTrackTraceLinks($order_id): array
 
         return $track_trace_links;
     }
+
+    /**
+     * @return int|null
+     * @throws Exception
+     */
+    public function getHighestShippingClass(): ?int
+    {
+        echo WCMP_Frontend::get_cart_shipping_class();
+        die();
```

Comment:
die() ?

Reply:
Ja anders komt er `0` achter. Het is een ajax functie (ik zal de functienaam even prefixen zodat dat duidelijk is)